src/backbone.associate.js

```diff
@@ -153,14 +153,18 @@
 
     var proto = klass.prototype;
 
-    if (!proto._associations) {
+    if (!proto.hasOwnProperty('_associations')) {
       // Patch initialize method in prototype
       _wrapMethod(proto, _initialize, 'initialize');
+      
+      // If an _associations namespace exists in the prototype chain, get it
+      var inherited_associations = proto._associations;
 
-      // Add namespace for associations
-      proto._associations = {};
+      // Add namespace for associations, adding in any inherited associations
+      proto._associations = _.extend({}, proto._associations, inherited_associations);
```

Comment:
If we change L168 instead, we should be able to skip the changes at L175 and L159...L164:

    // at L168
    proto._associations = _.extend({}, proto._associations, associations)